src/meta/src/hummock/compaction/picker/mod.rs

```diff
@@ -36,6 +39,8 @@ pub use ttl_reclaim_compaction_picker::{TtlPickerState, TtlReclaimCompactionPick
 
 use crate::hummock::level_handler::LevelHandler;
 
+pub const MAX_COMPACT_LEVEL_COUNT: usize = 42;
```

Comment:
Just curious, why is this set to 42?

Reply:
This is the value from some performance test